web/main/models.py

```diff
@@ -2155,11 +2155,11 @@ def export(self, include_annotations, file_type='docx', export_options=None, is_
             'export_options': export_options,
             'export_date': datetime.now().strftime("%Y-%m-%d"),
             'cloned_from': cloned_from,
-        })
+        }).replace('\xa0', '')
```

Comment:
This is extraneous and I removed it. Regarding the larger issue, I think they're both coming in through pastes, and TinyMCE adds \xa0 when a user types multiple spaces, too. I'm a bit surprised by that because I think \xa0 is specific to some encodings.  I think this should be an issue— but relatively low priority.

Reply:
Ah, cool beans.